language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/util/TestUtil.java

```diff
@@ -493,8 +493,19 @@ public static String getSemanticTokensResponse(Endpoint serviceEndpoint, String
      * @throws IOException Exception while reading the file content
      */
     public static void openDocument(Endpoint serviceEndpoint, Path filePath) throws IOException {
+        openDocument(serviceEndpoint, filePath, filePath.toUri().toString());
+    }   
+    
+    /**
+     * Open a document. Allows specifying the file URI to simulate different URI schemes.
+     *
+     * @param serviceEndpoint Language Server Service Endpoint
+     * @param filePath        Path of the document to open
```

Comment:
Missing documentation for parameter `fileUri`

Reply:
Done!